app/services/admin/work_report.rb

```diff
@@ -139,18 +139,22 @@ def statuses
     def work_forms
       @work_forms ||= druids.map do |druid|
         cocina_object = Sdr::Repository.find(druid:)
+        work = Work.find_by(druid:)
         # only cocina_object will be used; other arg assignments are irrelevant
-        Form::WorkMapper.call(cocina_object:, doi_assigned: true, agree_to_terms: true, version_description: nil,
-                              collection: nil)
+        work_form = Form::WorkMapper.call(cocina_object:, doi_assigned: true, agree_to_terms: true,
+                                          version_description: nil, collection: nil)
+        work_form.content_id = Contents::Builder.call(cocina_object:, user: work.user, work:).id
+        work_form
```

Comment:
Why all these shenanigans here to set the `content_id` in the work form?  Because if we need to use `Contents::Builder.call` to get the correct `content` model object, we need to have the cocina object to do this.  We already have it loaded here, so we might as well make use of it instead of loading it again when iterating below during the report.

Reply:
Though this approach has some potential performance considerations.  We build up an array of WorkForms in memory and then iterate over them below.  If we run a report over many many druids, which could potentially happen in the future, this could blow up.  Perhaps we should do this directly in the loop when generating the report instead of building it all in memory first.